### DISCUSSION TITLE
How to change the readthedocs button's behavior

### DISCUSSION THREAD

**Discussion by mendax1234** [pradyunsg/furo#470] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/62934294/176168972-8959198d-e58e-40ae-8205-08dbdf46f173.png)
l find that under the default furo theme, every time l move my cursor to the readthedocs badge , it will unfold . However, l don't want that because it is really annoying. l remember that the sphinx-rtd-theme has a button for it and only when l click the button, the bage will unfold. @pradyunsg 
<img width="246" alt="image" src="https://user-images.githubusercontent.com/62934294/176169325-3f6e037e-1148-4776-b7d8-c76683dd42b9.png">

**Comment by pradyunsg** [comment, marked as answer]
No, you can't change the behaviour.

**Reply by mendax1234** [reply]
> No, you can't change the behaviour.

But l found that the Photonvision docs's readthedocs badege is changed. (which is also built in furo theme) 
<img width="938" alt="image" src="https://user-images.githubusercontent.com/62934294/177004317-95314ad2-56b9-4523-8bd4-b6c5439fc026.png">

And here is their [link](https://docs.photonvision.org/en/latest/docs/examples/index.html)

**Reply by pradyunsg** [reply]
They haven't built with the current version of Furo.

**Reply by mendax1234** [reply]
> They haven't built with the current version of Furo.

Emmm , l'd like the readthedocs badge with a button which l can click to decide whether it will unfold or not. Can you make a customization function for it? Like creating an option in `conf.py` to let users choose which type they prefer. Thanks!

**Reply by pradyunsg** [reply]
I'm not going to be adding a configuration knob for this, no.

I'll explore if it makes sense to change the behaviour from on-hover to on-click, although to temper your expectations, it's quite possible that I prefer the status quo design and keep it this way.

**Reply by mendax1234** [reply]
> I'm not going to be adding a configuration knob for this, no.
> 
> I'll explore if it makes sense to change the behaviour from on-hover to on-click, although to temper your expectations, it's quite possible that I prefer the status quo design and keep it this way.

Well, thanks!
